src/signup/router.py

```diff
@@ -1,83 +1,63 @@
 from datetime import datetime
 
-import httpx
 from fastapi import APIRouter
 from pydantic import BaseModel
-from fastapi.responses import JSONResponse
 from src.core.dependencies import get_api_config
 
+from sqlalchemy import create_engine, String, DateTime, URL
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker, Mapped, mapped_column
+
 config = get_api_config()
 
 router = APIRouter()
 
+url_postgresql = URL.create(
+    drivername="postgresql",
+    username=config.DATABASE_USERNAME,
+    password=config.DATABASE_PASSWORD,
+    host=config.DATABASE_HOST,
+    port=5432,
+    database=config.DATABASE_NAME
+)
+engine = create_engine(
+     url_postgresql
+)
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
```

Comment:
ok it will work but don't we want to move engine creation to other file and use it when needed?
https://fastapi.tiangolo.com/tutorial/sql-databases/#create-the-database-tables
we could create `get_db` depdendency i our dependencies file and pass it even using FastAPI Depends
https://fastapi.tiangolo.com/tutorial/dependencies/

it will initialize the session for us if needed and we don't have to remember to close the connection